src/ui/components/Dashboard/DashboardViewerContent.js

```diff
@@ -107,8 +107,8 @@ function DashboardViewerContentHeader({
         <input type="checkbox" onChange={handleHeaderCheckboxClick} checked={selectedIds.length} />
       </th>
       <th>PREVIEW</th>
-      <th>TITLE</th>
       <th>PAGE TITLE</th>
+      <th>REPLAY</th>
```

Comment:
Let's keep the same column labels here for now until we decide what to do with titles.

Reply:
oh my bad forgot to undo that one, fixing now